ChangeLog

```diff
@@ -1,18 +1,64 @@
-Upcoming / Unreleased
-=====================
-- Added and updated some examples:
-  - Updated Extensions\AddHotelCallout
-  - Updated AdvancedOperations\AddSmartCampaign
-  - Added AdvancedOperations\AddResponsiveSearchAdWithAdCustomizer
-  - Added ShoppingAds\AddPerformanceMaxRetailCampaign
-  - Added AdvancedOperations\AddPerformanceMaxCampaign
-  - Added Remarketing\UploadConversionEnhancement
-  - Rename several examples
+11.0.0
+======
+- Added support for v10 of the Google Ads API.
+- Removed support for v7 of the Google Ads API.
+- Renamed Google My Business (GMB) to Business Profile everywhere.
 - Fix https://github.com/googleads/google-ads-dotnet/issues/331. Always get a refresh token in the
 web application example. Fix was contributed by Jhon Samamé
 (https://github.com/johnpisg)
-- Renamed Google My Business (GMB) to Business Profile everywhere.
-
+- Fixed https://github.com/googleads/google-ads-dotnet/issues/283. Read requestId from the
+ response object instead of response headers for streaming calls.
+- Added new code examples
+  - AdvancedOperations/AddCallAd
+  - AdvancedOperations/AddDynamicPageFeedAsset
+  - AdvancedOperations/AddPerformanceMaxCampaign
+  - AdvancedOperations/AddResponsiveSearchAdWithAdCustomizer
+  - Extensions/AddCall
+  - Remarketing/AddDynamicRemarketingAsset
+  - Remarketing/UploadConversionEnhancement
+  - Remarketing/UploadConversionWithIdentifiers
+  - ShoppingAds/AddPerformanceMaxProductListingGroupTree
+  - ShoppingAds/AddPerformanceMaxRetailCampaign
+- Updated code examples
+  - AccountManagement/GetAccountInformation
+  - AdvancedOperations/AddDisplayUploadAd
+  - AdvancedOperations/AddDynamicPageFeed
+  - AdvancedOperations/AddLocalCampaign
+  - AdvancedOperations/AddSmartCampaign
+  - AdvancedOperations/AddSmartDisplayAd
+  - BasicOperations/GetKeywords
+  - Extensions/AddAffiliateLocationExtensions
+  - Extensions/AddHotelCallout
+  - Misc/UploadImageAsset
+  - Remarketing/SetUpAdvancedRemarketing
+  - Remarketing/SetUpRemarketing
+- Renamed code examples
+  - Extensions/AddGoogleMyBusinessLocationExtensions => Extensions/AddGoogleMyBusinessLocationExtensions  
+Marked multiple issues as fixed.
```

Comment:
Should these be a bullet point (`-` prefix), and the list be indented, similar to the section on code examples?

Reply:
Done.